README.md

```diff
@@ -102,16 +102,69 @@ Server is running.
                 "Sid": "utapiMetrics",
                 "Action": [ "utapi:ListMetrics" ],
                 "Effect": "Allow",
-                "Resource": "arn:scality:utapi:::buckets/*"
+                "Resource": [
+                    "arn:scality:utapi::012345678901:buckets/*",
+                    "arn:scality:utapi::012345678901:accounts/*",
+                    "arn:scality:utapi::012345678901:users/*",
+                ]
             }
         ]
     }
     EOF
     ```
 
-    The user can also be restricted to only some buckets
-    (foo bucket for example) by changing the `Resource` property to
-    `"Resource": "arn:scality:utapi:::buckets/foo"`
+    In the above sample, the `Resource` property includes a series of Amazon
+    Resource Names (ARNs) used to define which resources the policy applies to.
+    Thus the sample policy applies to a user with an account ID '012345678901',
+    and grants access to metrics at the levels 'buckets', 'accounts', and
+    'users'.
+
+    The account ID of the ARN at any metric level can also be omitted, meaning
```

Comment:
The account ID of the ARN ... can also be omitted, allowing any account to access metrics for those resources. As an example, we can extend the above sample policy to allow any account to access metrics at the level 'service':